docs/user-guide.md

```diff
@@ -31,11 +45,22 @@ The partner can select the language of their preference from the dropdown in the
 * French
 * Arabic
 
+_Note_: Once a partner has registered, they must go into the partner management portal to access partner details.
```

Comment:
After registering, partners need to access the partner management portal to retrieve partner details.